lib/openstudio-workflow.rb

```diff
@@ -153,3 +153,73 @@ def load_adapter(name, adapter_options = {})
     end
   end
 end
+
+# Extend OS Runner to persist measure information throughout the workflow
+class ExtendedRunner < OpenStudio::Ruleset::OSRunner
+  # Allow former arguments to be set and read
+  #todo: Consider having the former arguments passed in in initialization, and define as attr_reader
+  attr_accessor :former_workflow_arguments
```

Comment:
Whoops spoke too soon, my previous comments only apply to virtual methods in C++ that you override in Ruby, looks like you are doing something different from what I did, we can chat today if that would help